apps/src/labs/projects/ProjectManager.ts

```diff
@@ -165,7 +176,7 @@ export default class ProjectManager {
   private enqueueSave() {
     this.saveQueued = true;
 
-    setTimeout(
+    this.timeoutId = window.setTimeout(
```

Comment:
Could we have a comment somewhere in this file explaining the timeout mechanism in general?